services/graphql-server/src/graphql/definitions/email/campaign.js

```diff
@@ -1,11 +1,80 @@
 const gql = require('graphql-tag');
 
 module.exports = gql`
+  extend type Query {
+    emailCampaign(input: EmailCampaignQueryInput!): EmailCampaign
+      @findOne(model: "email.Campaign", using: { id: "_id" })
+    emailCampaigns(input: EmailCampaignQueryInput!): EmailCampaignConnection
+  }
 
-type EmailCampaignTestRecipient {
-  firstName: String
-  lastName: String
-  email: String
-}
+  input EmailCampaignQueryInput {
```

Comment:
Your inputs should be at the end -- definitions should be ordered as:
1. special types (query, mutation)
2. enums
3. other types
4. inputs